AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyCommon.py

```diff
@@ -50,4 +49,4 @@ def expand_qt_container_rows(self, object_name: str) -> None:
         for child in children:
             check_box = child.findChild(QtWidgets.QCheckBox)
             if check_box and not check_box.isChecked():
```

Comment:
General Note: I've noticed you've done a few of the same checks in this code. Makes me think there may be a good use case for a UI conditional helpers util at a later point in time if QtPy gains in traction and there's a number of multiple conditional checks in the UI like this so a test writer doesn't need to know that they need to do this multiple conditional check.

At this time this doesn't need to happen, but something for us to keep an eye out for as we move forward.

Reply:
I think I have a quick solution to this. It will use this code in the common class but in the asset editor class it will be tidied up

Reply:
ok what I wanted to do works and makes the asset editor class a bit shorter